meetings/santiago2019/index.md

```diff
@@ -51,8 +51,8 @@ for a flavor of the workshop.
 **To register your attendance at the workshop, (and optionally submit a proposal
 for a talk), [use this form](https://goo.gl/forms/2nEljULHwC3zzcE13).**
 
-Talk proposals are also welcome until December 21st and talk acceptance will
-be notified by January 4th. 
+Talk proposals are welcome until <s>December 21st</s> **January 4th** and talk acceptance will be 
```

Comment:
According to https://en.wikipedia.org/wiki/Strikethrough, `<s>` was deprecated in 1999. Would markdown `~strikethrough~` work?

Reply:
`<s>` worked locally haha. Changed to `~~`.